traj:
  [
    [ #traj0
      [[1.1,2.2,3.3],[3.1,3.2,3.3]], #point0 [pos,vel]
      [[8.1,8.2,8.3],[9.1,9.2,9.3]] #point1 [pos,vel]
    ],
    [ #traj1
      [[5.1,5.2,5.3],[6.1,6.2,6.3]], #point0 [pos,vel]
      [[8.1,8.2,8.3],[9.1,9.2,9.3]], #point1 [pos,vel]
      [[3.1,3.2,3.3],[3.1,3.2,3.3]] #point2 [pos,vel]
    ]
  ]
  

ModIds: [1.0,2.0,3.2,4.1]
pos: [12.0,23.2,34.3,45.4]
traj1:
  point0:
    pos: [1.0,2.2,3.1,4.0]
    vel: [11, 22, 33, 44]
  point1:
    pos: 5 6 7
    vel: 55 66 77
traj2:
  point0:
    pos: 1 2 3 4
    vel: 11 22 33 44
  point1:
    pos: 5 6 7 8
    vel: 55 66 77 88
  point2:
    pos: 8 9 7 9
    vel: 55 66 77 99
